types/src/bytesrepr.rs

```diff
@@ -116,6 +116,7 @@ pub fn allocate_buffer<T: ToBytes>(to_be_serialized: &T) -> Result<Vec<u8>, Erro
 #[derive(Copy, Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
 #[cfg_attr(feature = "datasize", derive(DataSize))]
 #[cfg_attr(feature = "json-schema", derive(JsonSchema))]
+#[cfg_attr(feature = "json-schema", schemars(rename = "SerializationError"))]
```

Comment:
I think maybe `BytesreprError` is more meaningful as it could be an error during deserialization too.  (Also, this line could be merged with the one above)

Reply:
done